internal/shim-sgx/src/lib.rs

```diff
@@ -29,14 +29,7 @@ pub const ENCL_SIZE_BITS: u8 = 31;
 /// FIXME: doc
 pub const ENCL_SIZE: usize = 1 << ENCL_SIZE_BITS;
 
-const XFRM: Xfrm = Xfrm::from_bits_truncate(
-    Xfrm::X87.bits()
-        | Xfrm::SSE.bits()
-        | Xfrm::AVX.bits()
-        | Xfrm::OPMASK.bits()
-        | Xfrm::ZMM_HI256.bits()
-        | Xfrm::HI16_ZMM.bits(),
-);
+const XFRM: Xfrm = Xfrm::from_bits_truncate(Xfrm::X87.bits() | Xfrm::SSE.bits());
```

Comment:
Could you motivate this change? Perhaps we want to add a comment about it here as well?